src/openMVG/dataio/FeedProvider.hpp

```diff
@@ -19,7 +19,7 @@ class FeedProvider
 {
 public:
   
-  FeedProvider(const std::string &feedPath, const std::string &calibPath);
+  FeedProvider(const std::string &feedPath, const std::string &calibPath = "");
```

Comment:
i know there was no doc, my bad.
It' better to tell the supported mode:
- direcotry
- sfm file
- txt file
- regex